src/server/scripts/Kalimdor/zone_feralas.cpp

```diff
@@ -207,8 +207,8 @@ class npc_shay_leafrunner : public CreatureScript
                 me->SetFaction(FACTION_ESCORT_N_NEUTRAL_ACTIVE);
                 me->GetMotionMaster()->MoveFollow(player, 3.f, M_PI);
 
-                _events.ScheduleEvent(EVENT_WANDERING_START, urand(40 * IN_MILLISECONDS, 70 * IN_MILLISECONDS));
-                _events.ScheduleEvent(EVENT_CHECK_FOLLOWER, 30 * IN_MILLISECONDS);
+                _events.ScheduleEvent(EVENT_WANDERING_START, 40s, 70s);
```

Comment:
`urand(x,y)` is the same to `x,y` ? you are adding a third parameter to `ScheduleEvent` is it the same?

Reply:
> `urand(x,y)` is the same to `x,y` ? you are adding a third parameter to `ScheduleEvent` is it the same?

Yes.